### PR TITLE
feat(Panel): Implement resizable panel

### PR DESCRIPTION
**Pull request by bdauria**
This allows the panel width to be resized:

resizable

### CODE REVIEW

**Comment by Thebarda** [comment]
Maybe, add a small block wih an icon on the left side of the panel might be a good idea. It tells directly to the user that the panel is resizable

**Review by Thebarda — APPROVED** [review]

**Comment by bdauria** [comment]
> Maybe, add a small block wih an icon on the left side of the panel might be a good idea. It tells directly to the user that the panel is resizable

What I see most of the time when areas are resizable, is that the only indicator is the 'resize' cursor while hovering the border. wdyt?

**Comment by Thebarda** [comment]
> > Maybe, add a small block wih an icon on the left side of the panel might be a good idea. It tells directly to the user that the panel is resizable
> 
> What I see most of the time when areas are resizable, is that the only indicator is the 'resize' cursor while hovering the border. wdyt?

The simple cursor seems OK to me by the way :)

File: src/Panel/index.tsx [hunk 1]
```diff
@@ -22,6 +23,8 @@ const useStyles = makeStyles<Theme, StylesProps>((theme) => ({
     height: '100%',
     display: 'grid',
     gridTemplate: 'auto auto 1fr / 1fr',
+    overflow: 'hidden',
+    width: ({ width }) => width,
```
Comment (jdelpierre) [inline src/Panel/index.tsx:27]: Why this syntax ?

Comment (bdauria) [reply src/Panel/index.tsx:27]: The width comes from the props, which are passed to the useStyles function. We can use it to create dynamic styles like we do here, to fix the width to the one coming from the props.

Comment (jdelpierre) [reply src/Panel/index.tsx:27]: Oh yes, I didn't pay attention to the useStyles() line (already present) which took "width" as a parameter.

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**